wurst/objects/items/MixingPot/OmnicurePotion.wurst

```diff
@@ -0,0 +1,19 @@
+package OmnicurePotion
+
+import Items
+import PotionsDefinition
+import LocalObjectIDs
+import ObjectIds
+import Omnicure
+import ClosureEvents
+import LocalAssets
+
+constant ICON = ICON_PATH.format("MinorRejuvPotion")
+constant MODEL = LocalModelItems.potionPurpleSmall
+constant TT = "This potion is said to be able to cure all allies."
+
+@compiletime function createOmnicurePotion()
+    new PotionDefinition(ITEM_OMNICURE, ICON, MODEL, "Omnicure Potion", "S", TT, 22, commaList(ABILITY_OMNICURE_ITEM), 0.70, commaList(ABILITY_OMNICURE_ITEM), 120)
+
+init
+    registerSpellEffectEvent(ABILITY_OMNICURE_ITEM, function castOmnicure)
```

Comment:
This really should need to depend on the other `Omnicure` package directly. Use a dummy caster.

Reply:
If the casting is currently a problem, then I think the solution should be the same as Cure All, where we delay the recursive `InstantDummyCaster` usage.

Reply:
> 
> 
> This really should need to depend on the other `Omnicure` package directly.

I don't think I understand this sentence.

Reply:
Oops, I meant should not. Just use a dummy caster so that the potion casts the real omnicure.